cloudbuild-version.yaml

```diff
@@ -0,0 +1,35 @@
+steps:
+- name: gcr.io/cloud-builders/gcloud
+  entrypoint: 'bash'
+  args: [ '-c', "gcloud secrets versions access latest --secret=github-token-access --format='get(payload.data)' | tr '_-' '/+' | base64 -d > token.txt" ]
```

Comment:
Add a comment about why we do this plz

Reply:
👍

Reply:
Thanks - I'm also curious about the `tr '_-' '/+'` bit